pkg/client/cli/telepresence_test.go

```diff
@@ -171,11 +173,13 @@ func (ts *telepresenceSuite) TearDownSuite() {
 }
 
 func (ts *telepresenceSuite) TestA_WithNoDaemonRunning() {
-	ts.Run("Version", func() {
-		stdout, stderr := telepresence(ts.T(), "version")
-		ts.Empty(stderr)
-		ts.Contains(stdout, fmt.Sprintf("Client: %s", client.DisplayVersion()))
-	})
+	if goRuntime.GOOS != "windows" {
+		ts.Run("Version", func() {
+			stdout, stderr := telepresence(ts.T(), "version")
+			ts.Empty(stderr)
+			ts.Contains(stdout, fmt.Sprintf("Client: %s", client.DisplayVersion()))
+		})
+	}
```

Comment:
Why can't this test run on Windows?

Reply:
it actually can now, we just had to update the stable.txt file; good catch, i'll re-enable this test

Reply:
And even then, it would be better to call `.SkipNow()` inside of the test.